Public/Src/Cache/ContentStore/Distributed/NuCache/RocksDbContentLocationDatabase.cs

```diff
@@ -195,7 +195,12 @@ private BoolResult Load(OperationContext context, StoreSlot activeSlot, bool cle
                 Tracer.Info(context, $"Creating rocksdb store at '{storeLocation}'.");
 
                 var possibleStore = KeyValueStoreAccessor.Open(storeLocation,
-                    additionalColumns: new[] { nameof(Columns.ClusterState), nameof(Columns.Metadata) }, rotateLogs: true);
+                    additionalColumns: new[] { nameof(Columns.ClusterState), nameof(Columns.Metadata) },
+                    rotateLogs: true,
+                    failureHandler: failure => {

```

Comment:
Formatting:)

Reply:
BTW, based on this line of code, the accessor actually throws unknown exception but not swallowing it: [http://index/?leftProject=BuildXL.KeyValueStore&leftSymbol=opeuwx8lnhai&file=KeyValueStoreAccessor.cs&line=601](http://index/?leftProject=BuildXL.KeyValueStore&leftSymbol=opeuwx8lnhai&file=KeyValueStoreAccessor.cs&line=601)


---
In reply to: [322401909](https://github.com/microsoft/BuildXL/pull/849#discussion_r322401909) [](ancestors = 322401909)

Reply:
It doesn't swallow it, but we can swallow it sometimes. It is also not straightforward to search for that error, while this one is more explicit

---
In reply to: [322403543](https://github.com/microsoft/BuildXL/pull/849#discussion_r322403543) [](ancestors = 322403543,322401909)